Woof/WoofTests/Model/Preferences/PreferencesHandlerSaveTests.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+
+final class PreferencesHandlerSaveTests: XCTestCase {
```

Comment:
The name of the file doesn't match the method name `set()`.

Reply:
fixed